src/functions/unreferencedReusableObject.ts

```diff
@@ -0,0 +1,29 @@
+import { IFunction } from '../types';
+import { isObject } from '../utils';
+
+export type OasReusableObjectsLocation = '#/definitions' | '#/components/schemas';
+
+export const unreferencedReusableObject: IFunction<{ reusableObjectsLocation: OasReusableObjectsLocation }> = (
```

Comment:
```suggestion
export const unreferencedReusableObject: IFunction<{ reusableObjectsLocation: string }> = (
```
We should allow any string to be passed.

Reply:
Fixed. I added some slight checking regarding the content though. Cool with you?